examples/viewer.py

```diff
@@ -0,0 +1,152 @@
+from magnum import gl, shaders
+from magnum.platform.glfw import Application
+
+import habitat_sim
+
+settings = {
+    "max_frames": 10,
+    "width": 256,  # Spatial resolution of the observations
+    "height": 256,
+    "scene": "data/scene_datasets/habitat-test-scenes/apartment_1.glb",  # Scene path
+    "default_agent": 0,
+    "sensor_height": 1.5,  # Height of sensors in meters
+    "rgb": True,  # RGB sensor
+    "seed": 1,
+    "enable_physics": True,
+    "physics_config_file": "data/default.phys_scene_config.json",
+    "silent": False,
+    "num_objects": 10,
+    "compute_shortest_path": False,
+    "compute_action_shortest_path": False,
+    "save_png": True,
+    "render_to_ui": True,
+}
+
+navmesh_settings = habitat_sim.NavMeshSettings()
+navmesh_settings.set_defaults()
+navmesh_settings.agent_radius = 0.1
+navmesh_settings.agent_height = 0.5
+navmesh_settings.agent_max_climb = 0.0
+navmesh_settings.agent_max_slope = 0.0
+
+
+def make_cfg(settings):
```

Comment:
Why can't this be reused from examples/settings?

Reply:
Maybe we want to migrate some of that from examples to utils instead and flesh it out a bit. I don't like the idea of importing from examples into utils. 🤔